vm/vm.go

```diff
@@ -296,6 +322,14 @@ func (vm *Engine) validateStateTransition(virtualTx *wire.MsgTx) error {
 			if err != nil {
 				return err
 			}
+		case asset.ScriptV1:
```

Comment:
Main motivation here is that `vm.ValidateWitnessV0` relies on constructing the virtual transaction with single input / single output. We can't break that now

Reply:
See above. We can't unconditionally make the new tree.